plutus-book/doc/01-preface.adoc

```diff
@@ -18,3 +18,8 @@ and understand the basics of Haskell programming prior to reading this book.
 
 Come and explore the world of Plutus and learn our new language through
 real life scenarios and sample code snippets!
+
+*Disclaimer:*
+
+The Writing Reliable Smart Contracts in Plutus eBook is the property of IOHK. All content and materials have been produced by IOHK and can be reused in accordance with the Apache 2.0 license. The exercises and code samples included in this book are examples and are used as learning aids. The source files for this eBook are stored in the IOHK Plutus GitHub repository and are covered by our open source license agreement. 
```

Comment:
Use `{doctitle}` instead of writing out the book title. Use `{orgname}` instead of IOHK. Maybe link to the Plutus respository with `{repo}[link text]`?